### ISSUE TITLE
Should Stealth Fighter no longer require General Promotion?

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#643]
Should USA Nighthawk no longer require General Promotion? Proposed by Logica.

Apparently this would help USA vs GLA in late game, likely vs Buggies, Busses and Jarmen.

Does USA need such a buff?

**USA Airforce** has Nighthawk by default.
**USA** has Nighthawk as Level 1 Promotion Point.
**USA Laser** has Nighthawk as Level 1 Promotion Point.
**USA SWG** has Nighthawk as Level 1 Promotion Point.

### Proposal 1
Remove Level 1 Promotion requirement.

### Proposal 2
Reduce build cost.

**Comment by ExiLe-ZH** [comment]
Maybe would help laser since it lacks longrange units, which sucks late game vs mass buggies. Besides directly targetting buggies it might also help with denying GLA's Tunnel Network System with Bunker Busters > slightly nerfing GLA's mobility.

Maybe SW aswell to make the combo move with Alpha's more meta (timing it on a tunnel with bunker busters).

**Comment by ZekeDlyoung** [comment]
I don't play that much, so I need more detail as to how a removing a Rank 1 GP helps in _late_ game.

I assumed the purpose was to have the Stealth Fighter available in the early game? Unless you're talking about not being able to get the Stealth FIghter in the late game because you've spent your points on other things already. In that case the question I have is would having one extra GP point justify having the SF available from the start?

EDIT: That said, maybe the Aurora should be GP locked instead of the SF

**Comment by Stubbjax** [comment]
> I don't play that much, so I need more detail as to how a removing a Rank 1 GP helps in _late_ game.
> 
> I assumed the purpose was to have the Stealth Fighter available in the early game? Unless you're talking about not being able to get the Stealth FIghter in the late game because you've spent your points on other things already. In that case the question I have is would having one extra GP point justify having the SF available from the start?

The GP is seldom purchased as it is very rarely worth it. It's simply not viable to use Stealth Fighters in the early-game due to the cost and setup time. And even then, it doesn't make sense as an early-game unit as it is not what they excel at. Stealth Fighters are very specialised anti-defence units, and when players already have other units that can effectively do the same job, it doesn't make much sense to spend a GP to get them.
> 
> EDIT: That said, maybe the Aurora should be GP locked instead of the SF

This would make more sense, though standard Auroras are already underused without the GP requirement.

**Comment by ReLaX82** [comment]
You are right, most things are underused as other units do the job in the state of the game (mid to late game.)

Just think about your switch idea, but would be drastical.

Hope Logica jumps in and leave more input what he thinks.

**Comment by xezon** [comment]
I think 1 Nighthawk attack can take out one Rocket Buggy. This makes them useful in Lategame. And being able to clear Tunnels.

**Comment by MTKing4** [comment]
They're more useful mid to late game, most popular usage of it in current meta is combo-ed with Bunker Buster upgrade + good firepower unit to destroy and entire tunnel with all its content, would be useful when this is more accessible and not just for airforce general Since they don't need a GP there

**Comment by Jundiyy** [comment]
Whilst I agree it would help and I am very sure more players would use it if it was free, I don't think it's something that needs to be changed. Other than the GP, it's also costly, of course with the changes we made it was cheaper but I would think players just naturally have a habit not to use it now (most of the time).
Also the fact that they'd rather spend the GP point on something else.

**Comment by MTKing4** [comment]
> Whilst I agree it would help and I am very sure more players would use it if it was free, I don't think it's something that needs to be changed. Other than the GP, it's also costly, of course with the changes we made it was cheaper but I would think players just naturally have a habit not to use it now (most of the time). Also the fact that they'd rather spend the GP point on something else.

If you keep the GP, and change cost/BT it will remain useless, this is not reasonable At All it's more of a disadvantage when you get it due to losing Spy drone, it has already been said, Cost/BT shouldn't be your solution to every problem.

**Comment by Jundiyy** [comment]
It's not about the only solution, if it's not needed, we shouldn't move away from the original game. This is something desirable. 
Stealth Fighters are used but not often. So that's why it's not a must to change them. 
Stealth Fighters with Auroras for example. Or just to pop open a tunnel. So it's not a must.

Most players just naturally go for Spy Drone (and half don't even place them properly and arw shot down in the first minute or so), later on they go for the powerful GPs and then when needing a plane they can't. They don't always need to go for an A10 for example. SF + Rockvees or Auroras also works in place.

**Comment by MTKing4** [comment]
If you have the chance to improve the game and not piss off the playerbase why shy away?

If that's your concern we can add it to a survey question and see what people say about this.

**Comment by Stubbjax** [comment]
> It's not about the only solution, if it's not needed, we shouldn't move away from the original game.
> 

Changes must also be weighed against the impact / risk - which I would argue would be very low here. Boosting strategic diversity by reducing the disadvantage incurred from using specialised units - without making them so much more viable that they change the core gameplay - can only be a good thing.

**Comment by ExiLe-ZH** [comment]
> They're more useful mid to late game, most popular usage of it in current meta is combo-ed with Bunker Buster upgrade + good firepower unit to destroy and entire tunnel with all its content, would be useful when this is more accessible and not just for airforce general Since they don't need a GP there

This is indeed it’s best use, either combining it with Alpha Aurora or mass Commie.

SW however often struggles with money and map control vs pro GLA’s, so usually the SF is too expensive already.

With Laser/USA in high level 1v1 you do actually sometimes see a Commie spam transition (vs Demo usually) but since this is already pretty niche you will not ever see the SF combo like ever.

For other uses like I mentioned in my previous post it’s surely not worth spending a GP either. 

Unlike the Marauder case, I think unlocking SF by default for all USA’s might be a positive change without any balance implications.

**Comment by xezon** [comment]
Same as Raptor, the Nighthawk is not used much. If we can make it more appealing to mix units, it may be good. Right now USA just needs Humvees + Missile Defenders + Snipers to win any combination of matches, other than vs Airforce.

**Comment by ZekeDlyoung** [comment]
> This would make more sense, though standard Auroras are already underused without the GP requirement.

Is it because Auroras are underused, or because people ban them in games :P 

I don't mind removing SF unlock either, as long as the extra GP doesn't cause any unintended gameplay changes.

I actually use the SF a lot more than the Raptor, but then I am n00b.

**Comment by DoMiNaToRuk2** [comment]
Yes remove the gen point promotion

**Comment by ExiLe-ZH** [comment]
So what we do here? 

I think removing Genpoint is good, but to keep some differentiation from air we could set it's requisite to Strat Center.

Otherwise it's not really a special unit whatsoever anymore.

**Comment by Stubbjax** [comment]
> I think removing Genpoint is good, but to keep some differentiation from air we could set it's requisite to Strat Center.

This sounds good to me.
